packages/core-components/package.json

```diff
@@ -36,6 +36,7 @@
     "@backstage/config": "workspace:^",
     "@backstage/core-plugin-api": "workspace:^",
     "@backstage/errors": "workspace:^",
+    "@backstage/plugin-catalog": "workspace:^",
```

Comment:
I believe core packages cannot depend on plugin packages, otherwise, it could introduce cyclical dependency.
Let me double-check with @backstage/maintainers.